tests/test_load_state.py

```diff
@@ -1,2 +1,67 @@
-class LoadStateTestCase():
-    pass
+import os
+import unittest
+
+from models import model
+
+
+class LoadStateTestCase(unittest.TestCase):
+    """
+    Testing strategy
+    partitions:
+        input:Previous state DB, path to DB, Correct content type, bad content,
+               non-existant DB, bad input type(non-string)
+        special case: Over write existing Internal state, Handle Large file,
+                      Missing Fields in DB, Already Loaded Data,
+                      Append DB state to internal state
+
+        This should cover the full catersian plane
+    """
+    dojo = model.Dojo("Andela-Kenya1")
+    dojo1 = model.Dojo("Andela-Kenya2")
+    def setUp(self):
+        self.dojo = LoadStateTestCase.dojo
```

Comment:
this import statement should be `dojo.LoadTestCase` also there is no `LoadTestCase`method in your model file.